content/blog/2022/11/ui-1.0.0-release.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Kubewarden UI 1.0.0 is released!
+authors:
+- Jordon Leach 
+date: 2022-11-01 # TODO: Update date before publishing
+---
+
+We are excited to announce that the Kubewarden UI 1.0.0 has been released!
+
+The UI is an Extension for [Rancher Manager](https://github.com/rancher/rancher/), now you will be able to enable Kubewarden policies for your Kubernetes clusters with a streamlined user experience.
+
+Let's take a look at what is achievable from the UI as a Cluster Admin.
```

Comment:
I would like to mention that it can install Kubewarden through the UI/Rancher resources too (with some example screenshot).

Reply:
I added an installation section along with a quick video.